Source: Timo Gurr <tgurr@exherbo.org>
Upstream: No
Reason: tst-resolv-binary runs into an endless loop with sydbox

diff -Naur glibc-2.27/resolv/Makefile glibc-2.27.new/resolv/Makefile
--- glibc-2.27/resolv/Makefile	2018-02-01 17:17:18.000000000 +0100
+++ glibc-2.27.new/resolv/Makefile	2018-02-02 13:32:28.466646055 +0100
@@ -51,7 +51,6 @@
   tst-res_hnok \
   tst-res_use_inet6 \
   tst-resolv-basic \
-  tst-resolv-binary \
   tst-resolv-edns \
   tst-resolv-network \
   tst-resolv-res_init-multi \
@@ -160,7 +159,6 @@
 $(objpfx)tst-bug18665: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-res_use_inet6: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-basic: $(objpfx)libresolv.so $(shared-thread-library)
-$(objpfx)tst-resolv-binary: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-edns: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-network: $(objpfx)libresolv.so $(shared-thread-library)
 $(objpfx)tst-resolv-res_init: $(libdl) $(objpfx)libresolv.so
